lunar/distribution.yaml

```diff
@@ -3107,6 +3107,29 @@ repositories:
       url: https://github.com/ros-perception/vision_opencv.git
       version: kinetic
     status: maintained
+  vision_visp:
+    doc:
+      type: git
+      url: https://github.com/lagadic/vision_visp.git
+      version: lunar
+    release:
+      packages:
+      - vision_visp
+      - visp_auto_tracker
+      - visp_bridge
+      - visp_camera_calibration
+      - visp_hand2eye_calibration
+      - visp_tracker
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/lagadic/vision_visp-release.git
+      version: 0.10.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the instructions to provide the necessary access to the @ros-pull-request-builder user?